### DISCUSSION TITLE
设定好了抢票时间，但是程序未启动抢票

### DISCUSSION THREAD

**Discussion by zhuhaochen001026** [mikumifa/biliTickerBuy#315] [提问专区]
设的中午12：00：00，提前按了开始抢票，显示等待中，一直到12：00：00之后仍然是等待中，我再点了一下开始抢票才开始。
我确定自己设定的时间都是正确的，昨天这个脚本也确实帮我抢到票了。
请问一下大佬，这是怎么回事呢？

**Comment by mikumifa** [comment]
https://biaozhunshijian.bmcx.com/
测一下电脑时间误差

**Reply by zhuhaochen001026** [reply]
抢票开始前校准过，现在也测试了，都是没问题的，前面两天的抢票也没有出现这种问题，您有什么头绪嘛？

**Reply by SouthSeafireworks** [reply]
最好不要设时间，你可以提前10s手动开始点，我两回抢票都是提前1分钟点的，亲测可以抢到，账号也没有被封（至少这两天没事）

**Reply by CinnabarCHU** [reply]
我提前2s都被封了55555
